### PR TITLE
Chore/add unit tests for vlmd-submission-tools

### PR DESCRIPTION
**Pull request by george42-ctds**
Jira Ticket: [HP-1166](https://ctds-planx.atlassian.net/browse/HP-1166)

### Improvements

* Add unit tests for vlmd-submission-tools

### Deployment changes

[HP-1166]: https://ctds-planx.atlassian.net/browse/HP-1166?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/ci_vlmd.yaml [hunk 1]
```diff
@@ -2,13 +2,12 @@ name: CI Workflow
...
+      - vlmd-submission-tools/**
+      - .github/workflows/ci_vlmd.yaml
   pull_request:
-    branches:
-      - master
+    paths:
+      - vlmd-submission-tools/
 
 jobs:
   test:
```
Comment (mfshao) [inline .github/workflows/ci_vlmd.yaml:13]: nitpicking: should consider change this job's name to something like `vlmd-wf-test` so easier to infer from the status page what this action iis testing for

**Review by mfshao — DISMISSED** [review]
lgtm 👍 one miinor comment

File: .github/workflows/ci_vlmd.yaml (hunk 1 above)
Comment (george42-ctds) [reply .github/workflows/ci_vlmd.yaml]: Good idea!

Comment (mfshao) [reply .github/workflows/ci_vlmd.yaml]: hmmm does not see the job get triggered after new commit 🤔

Comment (george42-ctds) [reply .github/workflows/ci_vlmd.yaml]: Oh, interesting. I think the test got triggered for the [commit for the name-change](https://github.com/uc-cdis/containers/actions/runs/6473862155). The next commit was for a conflict in the root-level secrets.baseline file. Other tests were triggered on that commit. I did another commit in the vlmd-submission-tools. It [triggered the ci_vlmd](https://github.com/uc-cdis/containers/actions/runs/6474317608).

Comment (mfshao) [reply .github/workflows/ci_vlmd.yaml]: that is right! thanks

**Review by mfshao — APPROVED** [review]

**Pull request merged**